src/main/java/icbm/classic/content/blocks/launcher/base/TileLauncherBase.java

```diff
@@ -173,11 +187,12 @@ public <T> T getCapability(Capability<T> capability, @Nullable EnumFacing facing
         if (capability == CapabilityItemHandler.ITEM_HANDLER_CAPABILITY)
         {
             return (T) inventory;
-        } else if (capability == ICBMClassicAPI.MISSILE_HOLDER_CAPABILITY)
+        }
+        if (capability == ICBMClassicAPI.MISSILE_HOLDER_CAPABILITY)
```

Comment:
Removing the else-if doesn't change the code and splits the visual linking. Its better to include it to let other devs know the chain of ifs are related.